selfdrive/locationd/test/test_locationd.py

```diff
@@ -5,9 +5,11 @@
 import unittest
 import time
 from cffi import FFI
+import capnp
```

Comment:
this should go above the from, like this:
```suggestion
import capnp
from cffi import FFI
```

The import convention in openpilot is standard library and third party, then openpilot imports. within those, imports go first and froms after preferably in alphabetical order.